package.json

```diff
@@ -10,13 +10,19 @@
     "start": "nf start",
     "lint": "eslint --color *.js; exit 0;",
     "test:watch": "mocha -w",
-    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec"
+    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec",
+    "migrate": "node_modules/.bin/sequelize db:migrate",
+    "migrate:undo": "node_modules/.bin/sequelize db:migrate:undo",
+    "migrate:undo:all": "node_modules/.bin/sequelize db:migrate:undo:all",
```

Comment:
I don't understand the distinction between the `undo` and `undo:all` - can you please include information in an appropriate location that documents these actions?